cmd/coder/main.go

```diff
@@ -15,6 +20,48 @@ import (
 func main() {
 	rand.Seed(time.Now().UnixMicro())
 
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	// Add a custom SIGQUIT handler that outputs to stderr and a well-known file
+	// in the home directory. This also prevents SIGQUITs from killing the CLI.
+	go func() {
+		sigs := make(chan os.Signal, 1)
+		signal.Notify(sigs, syscall.SIGQUIT)
+		for {
+			select {
+			case <-ctx.Done():
+				return
```

Comment:
It won’t make much of a difference here, but as a practice we could unregister the signal handler to clean up after the routine.